src/Pages/Competition.jsx

```diff
@@ -2,14 +2,22 @@ import { useParams, useNavigate } from "react-router-dom";
 import { useEffect, useState } from "react";
 
 function Competition(props) {
-  const navigate = useNavigate();
-  function handleClick(event, ticket) {
-    event.preventDefault();
-    navigate(`/tickets`);
-  }
+
+    const navigate = useNavigate();
+    function handleClick (event, ticket) {
+      event.preventDefault()
+      navigate(`/tickets`)
+    }
+    
+    function handleClick2 (event, ticket) {
+      event.preventDefault()
+      navigate(`/register`)
+    }
+  
   const dogs = props.dogs;
   const competitions = props.competitions;
 
+
   console.log("props inside competition: ", props);
   console.log("dogs in competition: ", dogs);
   console.log("competitions in Competition", competitions);
```

Comment:
Might be a good idea to delete these console logs if you don't need them anymore :)